files/values.yaml.tftpl

```diff
@@ -2,26 +2,133 @@
 
 # Configuration for HA deployment of Kyverno
 # https://kyverno.io/docs/installation/methods/#high-availability-installation
```

Comment:
The two lines are no more relevant here. Move them in `variables.tf` file where we have the replicas configuration.